docker-compose-dev.yml

```diff
@@ -1,136 +1,79 @@
-# Example Wikibase docker-compose setup
 version: '3.4'
 
 x-common-variables: &wikibase_variables
-  DB_SERVER: mysql.svc:3306
-  MW_ADMIN_NAME: ${MW_ADMIN_NAME}
-  MW_ADMIN_PASS: ${MW_ADMIN_PASS}
-  MW_ADMIN_EMAIL: ${MW_ADMIN_EMAIL}
-  MW_WG_SECRET_KEY: ${MW_SECRET_KEY}
-  # Disable jobs running after requests when wikibase_jobrunner is defined
-  MW_WG_JOB_RUN_RATE: 0
-  DB_USER: ${DB_USER}
-  DB_PASS: ${DB_PASS}
-  DB_NAME: ${DB_NAME}
-  WIKIBASE_HOST: ${WIKIBASE_HOST}
-  QS_PUBLIC_SCHEME_HOST_AND_PORT: http://localhost:8840
-
-x-common-variables: &wikibase_extra_variables
-  MW_ELASTIC_HOST: ${MW_ELASTIC_HOST}
-  MW_ELASTIC_PORT: ${MW_ELASTIC_PORT}
-
+    WIKIBASE_HOST: localhost
+    WIKIBASE_PORT: 8080    
+    WDQS_FRONTEND_PORT: 8834
+    QUICKSTATEMENTS_HOST: localhost
+    QUICKSTATEMENTS_PORT: 8840
+    # required by quickstatements for creating OAuth consumers,
+    # and by wikibase for redirecting back to quickstatements
+    QS_PUBLIC_SCHEME_HOST_AND_PORT: 'http://${QUICKSTATEMENTS_HOST}:${QUICKSTATEMENTS_PORT}'
+    WB_PUBLIC_SCHEME_HOST_AND_PORT: 'http://${WIKIBASE_HOST}:${WIKIBASE_PORT}'
+    
 services:
   wikibase:
-    image: "${WIKIBASE_BUNDLE_IMAGE_NAME}"
-    #build: ./mediawiki
-    container_name: mardi-wikibase
-    links:
-      - mysql
-    depends_on:
-      - mysql
     ports:
       - "${WIKIBASE_PORT}:80"
-    volumes:
-      ## This shares the configuration with jobrunner
-      - shared:/var/www/html/:rw
-      - ./mediawiki/LocalSettings.d:/var/www/html/LocalSettings.d
-      - ./mediawiki/images/MaRDI_Logo.png:/var/www/html/images/MaRDI_Logo.png
-      - ./mediawiki/images/favicon.png:/var/www/html/images/favicon.png
-      - ./mediawiki/extensions/JsonConfig:/var/www/html/extensions/JsonConfig
-      - ./mediawiki/extensions/TemplateStyles:/var/www/html/extensions/TemplateStyles
-      - ./mediawiki/extensions/Math:/var/www/html/extensions/Math
-      - ./mediawiki/extensions/MathSearch:/var/www/html/extensions/MathSearch
-      - quickstatements-data:/quickstatements/data
-      - ./mediawiki/extra-install.sh:/extra-install.sh
-    networks:
-      default:
-        aliases:
-         - ${WIKIBASE_HOST}
-         - wikibase-docker.svc
+    restart: 'no'
```

Comment:
how are extensions and additional files added now, if volumes are removed ?

Reply:
wen you call docker-compose up, you call docker-compose.yml and docker-compose-dev.yml (see README). So the extensions etc. are added to docker-compose. Is that acceptable?